cypress/support/fragments/bulk-edit/bulk-edit-actions.js

```diff
@@ -256,16 +256,33 @@ export default {
     cy.wait(500);
   },
 
-  verifyUsersActionDropdownItems(isDisabled = false) {
+  verifyUsersActionDropdownItemsInCaseOfError() {
     cy.expect([
-      dropdownMenu.find(Checkbox({ name: 'Active', checked: true, disabled: isDisabled })).exists(),
-      dropdownMenu.find(Checkbox({ name: 'Last name', checked: true, disabled: isDisabled })).exists(),
-      dropdownMenu.find(Checkbox({ name: 'First name', checked: true, disabled: isDisabled })).exists(),
-      dropdownMenu.find(Checkbox({ name: 'Barcode', checked: true, disabled: isDisabled })).exists(),
-      dropdownMenu.find(Checkbox({ name: 'Patron group', checked: true, disabled: isDisabled })).exists(),
-      dropdownMenu.find(Checkbox({ name: 'Username', checked: true, disabled: isDisabled })).exists(),
-      dropdownMenu.find(Checkbox({ name: 'Email', checked: false, disabled: isDisabled })).exists(),
-      dropdownMenu.find(Checkbox({ name: 'Expiration date', checked: false, disabled: isDisabled })).exists(),
+      DropdownMenu().find(Checkbox('Username')).has({ disabled: true }),
```

Comment:
Remove to const all repeating items from dropdown menu
const usernameCheckbox = Checkbox('Username'); for example